core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.mainpage.html

```diff
@@ -2,6 +2,11 @@
 <html ng-app="oppia" itemscope itemtype="http://schema.org/Organization">
   <head>
     @load('base-components/header.template.html', {"title": "Contributor Dashboard - Oppia"})
+    <style>
+      .headroom--unpinned {
+        transform: translateY(0);
+      }
+    </style>
```

Comment:
when we scroll through view on the contributor dashboard page navbar goes up due to global CSS.
.headroom--unpinned {
    transform: translateY(-100%);
  }

earlier we are overriding this on file `contributor-dashboard-page.component.html` 
.headroom--unpinned {
    transform: translateY(0%);
  }

Reply:
Why doesn't the previous solution in `core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.html` work anymore?

Reply:
In angular css scope will remain for that component only so as to change the CSS property of the navigation bar in the contributor-dashboard-page. we have to declare this in a global space.

Reply:
I see. So shouldn't we also remove the equivalent style in `core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.html`?

Reply:
done